daemon/daemon_main.go

```diff
@@ -918,6 +913,39 @@ func runDaemon() {
 		}
 	}
 
+	api := d.initializeAPI()
```

Comment:
Why isn't the `initializeAPI()` _initializing_ the API?

Reply:
I don't follow. It creates the `api` structure. Would you prefer `instantiateAPI`?

Reply:
Yes, or move the next lines inside the function. The following lines are the ones that are actually initializing the API

Reply:
Will be called `instantiateAPI` in next upload.